lib/logger.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+/**
+ * Standard loggers (with prefixes so they're easy to search for)
```

Comment:
so handy! http://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-logging.html

Reply:
Yeah, and what they don't tell you is that it's pretty much impossible to find your log statements in CW without some obvious prefix like `[INFO]`.  Too much other noise in there, as it logs every lambda execution, how much memory/time it used, how much you were billed for, etc.

Reply:
:no_mouth: